3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -84,6 +84,70 @@ Per i requisiti di sistema e hardware si rimanda al documento #st_v.
 
 
 = Istruzioni d'uso
+== Avvio e Configurazione dell'ambiente
+
+Il software all'avvio si presenta come segue:
+#figure(
+  image("./imgs/avvio.png", width: 60%),
+  caption: [
+    Avvio del software
+  ],
+) <avvio>
+qui vengono proposte due diverse configurazioni iniziali possibili, *Planimetria rettangolare* e *Planimetria personalizzata*. Le differenze tra le due modalità di lavoro verranno descritte nei capitoli dedicati.
+
+Si può scegliere una modalità di lavoro selezionandola nel pannello di @avvio e premendo il pulsante *Prossimo* in basso a destra.
+
+=== Inizializzazione Planimetria Rettangolare
+
+#figure(
+  image("./imgs/planimetria_rettangolare.png", width: 60%),
+  caption: [
+    Definizione parametri del piano rettangolare
+  ],
+) <piano_rettangolare>
+
+Selezionata la Planimetria Rettangolare nella schermata di @avvio il sistema mostrerà la schermata mostrata in @piano_rettangolare, dove è possibile definire la lunghezza e la profondità del magazzino che vogliamo creare.
```

Comment:
Cambierei il paragrafo in

"Selezionata l'opzione "Planimetria rettangolare" nella schermata di `@avvio`, verrà mostrata la schermata presente in `@piano_rettangolare`, dove sarà possibile definire la lunghezza e la profondità del magazzino che vogliamo creare."

(ho messo in monospace i tag solo perché sennò GitHub creava dei tag veri)